diff --git a/node_modules/expo-image-loader/android/build.gradle b/node_modules/expo-image-loader/android/build.gradle
index 12ef380..8479165 100644
--- a/node_modules/expo-image-loader/android/build.gradle
+++ b/node_modules/expo-image-loader/android/build.gradle
@@ -37,7 +37,7 @@ version = '3.2.0'
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier.set("sources")
   from android.sourceSets.main.java.srcDirs
 }
 
